sycl/test-e2e/Matrix/SPVCooperativeMatrix/SG32/joint_matrix_bf16_fill_k_cache.cpp

```diff
@@ -0,0 +1,24 @@
+//==--- joint_matrix_bf16_fill_k_cache.cpp  - DPC++ joint_matrix----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
+// UNSUPPORTED: gpu-intel-dg2
+// REQUIRES: gpu, aspect-ext_intel_matrix
```

Comment:
Please, clean-up `REQUIRES: gpu` that were not present in the original tests here and below
```suggestion
// REQUIRES: aspect-ext_intel_matrix
```

Reply:
Fixed here an in another cases